smdebug/tensorflow/keras.py

```diff
@@ -348,7 +349,10 @@ def _prepare_tensors_available_post_step(self):
 
                 # Add tensor to custom collections
                 for custom_coll in custom_collections:
-                    if match_inc(tensor_ref.name, custom_coll.include_regex):
+                    if (
+                        match_inc(tensor_ref.name, custom_coll.include_regex)
+                        and tensor_ref.tf_obj is not None
```

Comment:
why this check? was there an issue without it?

Reply:
tensor_ref.tf_obj is none for certain tensors (kernelGrad) and ends up throwing a warning